### ISSUE TITLE
checkers: fix commentedOutCode regression

### ISSUE THREAD

**Issue by quasilyte** [go-critic/go-critic#851]
For code below commentedOutCode now gives a warning:

```go
func ExampleFoo() {
	// Output:
	// blah
	// blah
}
```
Should handle such comments properly.
There are probably more regressions due to the recent multi-statement handling, but this issues is about example "output:" specifically.
